doc/languages-frameworks/cuda.section.md

```diff
@@ -54,3 +54,65 @@ for your specific card(s).
 
 Library maintainers should consult [NVCC Docs](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/)
 and release notes for their software package.
+
+## Adding a new CUDA release {#adding-a-new-cuda-release}
+
+> **WARNING**
+>
+> This section of the docs is still very much in progress. Feedback is welcome in GitHub Issues tagging @NixOS/cuda-maintainers or on [Matrix](https://matrix.to/#/#cuda:nixos.org).
+
+The CUDA Toolkit is a suite of CUDA libraries and software meant to provide a development environment for CUDA-accelerated applications. Until the release of CUDA 11.4, NVIDIA had only made the CUDA Toolkit available as a multi-gigabyte runfile installer, which we provide through `cudaPackages.cudatoolkit`. From CUDA 11.4 and onwards, NVIDIA has also provided CUDA redistributables (“CUDA-redist”): individually packaged CUDA Toolkit components meant to facilitate redistribution and inclusion in downstream projects. These packages are available in the `cudaPackages` package set.
```

Comment:
What do you think about re-iterating what these paths like `cudaPackages` are?

```diff
- which we provide through `cudaPackages.cudatoolkit`
+ which we provide through the `cudaPackages.cudatoolkit` attribute
```

or (I don't like this though)

```diff
- which we provide through `cudaPackages.cudatoolkit`
+ which we provide through the `cudaPackages.cudatoolkit` attribute (e.g. `nix-build '<nixpkgs>' -A cudaPackages.cudatoolkit`)
```

Reply:
Yes, let's use search.nixos.org links every time we have to reference an attribute

Reply:
Resolved in 38c103dddb4512497378c57960417ce575d0f2ea